### PR TITLE
Add accessibility page

### PR DESCRIPTION
**Pull request by theacodes**

### CODE REVIEW

File: content/accessibility/contents.lr [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+----------------
+
+PyCascades 2019 will be hosted in the University of Washington's Kane Hall. We will be using two auditoriums, KNE 130 and KNE 120, as well as the Walker-Ames room for food & beverage service. You can read UW's `ADA Access Guide`_ for these rooms for more details, but most of it is included on this page for convenience.
+
+.. _ADA Access Guide: https://www.washington.edu/admin/ada/kane.php
+
+Wheelchair access
+------------------
+
+We personally walked through the campus to ensure wheelchair accessibility. There are ramps, no steps, no steep slopes, no lips on doors, automatic doors, and elevators. Please take a look at the campus map below and take a look at the accessibility information provided by the venue.
```
Comment (ellezv) [inline content/accessibility/contents.lr]:
```suggestion
We personally walked through the campus to ensure wheelchair accessibility. There are ramps, no steps, no steep slopes, no lips on doors, automatic doors, and elevators outside of the building. Please take a look at the campus map below and take a look at the accessibility information provided by the venue.
```

**Review by cache-rules — APPROVED** [review]

**Review by ellezv — COMMENTED** [review]
Can we also add a section for the mother's room?
Otherwise everything looks good to me :)

**Comment by theacodes** [comment]
@ellezv I couldn't find any details about the mother's room, do you have link?

**Comment by theacodes** [comment]
Mother's room info added, merging.

**Pull request merged**